src/backend/joseki.be/webapp/Models/InfrastructureComponentSummary.cs

```diff
@@ -8,19 +8,19 @@ namespace webapp.Models
     public class InfrastructureComponentSummary
     {
         /// <summary>
-        /// The unique Id of a component.
+        /// The unique Id of a component summary.
         /// </summary>
         public string Id { get; }
```

Comment:
Probably, we do not need an Id here anymore

Reply:
removed the `Id` for `InfrastructureComponentSummary`